File: shoring/desk/src/main/java/com/panopset/marin/oldswpkg/games/blackjack/LayoutChips.java
package com.panopset.marin.oldswpkg.games.blackjack;

public class LayoutChips {

    public final int chipXnextBet;
    public final int chipXstack;
    public final int chipY;

    public LayoutChips(int left, int bottom, int chipWidth, int chipHeight) {
        chipXnextBet = left;
        chipXstack = left + chipWidth;
        chipY = bottom - ((int) (chipHeight * 1.5));
    }

}

File: code/shoring/desk/src/main/kotlin/com/panopset/marin/oldswpkg/games/blackjack/LayoutChips.kt
package com.panopset.marin.oldswpkg.games.blackjack

class LayoutChips(val chipXnextBet: Int, bottom: Int, chipWidth: Int, chipHeight: Int) {
    val chipXstack: Int
    val chipY: Int

    init {
        chipXstack = chipXnextBet + chipWidth
        chipY = bottom - (chipHeight * 1.5).toInt()
    }
}
